### ISSUE TITLE
Player is invisible on animation

### ISSUE THREAD

**Issue by kineese** [altmp/altv-issues#224]
When a player is doing an animation and someone is entering the players streaming-range the player is invisible for the entering player until the player ends the animation
## Expected Behavior
Player should be visible during animation, even for streaming-range entering players

## Current Behavior
Player is invisible

## Steps to Reproduce
Let a player do an animation. Let another player see the player and let him drive away for a long range and then let him come back.
https://plays.tv/video/5d49e36f4ebf3af4be/animation

**Comment by vadzz-dev** [comment]
Cannot reproduce

**Comment by BlackFire51** [comment]
Some of my players can reporduce it farly conistend. 7/7 arriving to our "sit" meeting yesterday hat that bug. I myself could not reporduce it yet.

Sniped of how the animations are controled 
`
function playAnimation(animDictionary, animationName, speed,flag){
	//alt.log("play ANim "+animDictionary)
	stopAnimation();
	requestAnimDictPromise(animDictionary).then(()=>{
		//alt.log("play anim NOW ",animDictionary, animationName)
		game.taskPlayAnim(game.playerPedId(), animDictionary, animationName, speed, 1, -1, flag, 1, false, false, false)
		activeDict=animDictionary;
		activeAnim=animationName;
		
	}).catch(()=>{alt.log('Promise returned reject anim')});
}

function getActiveAnim(){
	return {dict:activeDict,anim:activeAnim}
}
function stopAnimation(){
	alt.log("scriptID "+localPlayer.scriptID+"  pedId: "+PlayerID)
	if(activeDict==null || activeAnim == null) return;
	game.stopAnimTask(PlayerID, activeDict, activeAnim, 0);
	game.removeAnimDict(activeDict);


	game.clearPedTasks(localPlayer.scriptID)
	game.clearPedSecondaryTask(localPlayer.scriptID);
	activeDict=null;
	activeAnim=null;
}
`

**Comment by lameule123** [comment]
This bug is still present

**Comment by AixMC** [comment]
Same here. Seems to be a bit random. 20 people are in the same animation in the same area. Players coming from further away see a few, but not all.

**Comment by nxtVentix** [comment]
If the Player Jumps, then they are visible

**Comment by MalteCr** [comment]
I have found a player beneath the ground at my side. He was using a sit animation. As soon as he moved, he was visible for me at the surface again.

**Comment by Benni9323** [comment]
Push please!

**Comment by ThomasMarangoni** [comment]
**Information**
Some information about this bug, it seems like it's triggered if you play a looped animation longer then 5 minutes. The longer you play an animation, the higher chance you have to get this bug. The player gets teleported under the ground.

**Media**
coordinates (https://i.imgur.com/hrOO7Ly.png) where the player should be:
{"x":480.05780029296875,"y":-1313.730224609375,"z":30.278474807739258}

coordinates (https://i.imgur.com/Z5epRk7.png) where the player is:
{"x":478.7660827636719,"y":-1313.434814453125,"z":-197.6778106689453}

and a video where the players where sitting in an MLO above the tunnel:
https://www.youtube.com/watch?v=2O5ty3RdN6g

**Comment by ThomasMarangoni** [comment]
This Animation seems to trigger the bug **"anim@arena@celeb@flat@solo@no_props@" "slide_a_player_a"** when played with flag 0.

**Comment by Sushi1337** [comment]
We further investigated this issue and gathered some interesting information:
- It happens with nearly every looped animation where the player can't move.
- The player is NOT invisible. Only his z value on position is wrong.
- The player is on the correct position on the stream in event and will slowly "fall" down
![image](https://user-images.githubusercontent.com/39070261/95320221-77ed7000-0899-11eb-990f-85df240d44e6.png)
(this is just one StreamIn event of a player that plays a looped animation - the position gets written to console every 500ms)
- It does not happen every time, but as longer the animation is played, the more often it happens
![image](https://user-images.githubusercontent.com/39070261/95320412-bb47de80-0899-11eb-811c-b0baac9ea2cd.png)
(this is basically just a alt.log with the position on the StreamIn event with some delay)
- The position remains correct on serverside all the time. You can basically send the entity of the StreamIn event to the server and the server will still have the correct position.
- We were not able to reproduce this issue with animations where the player can move (e.g. with the upperBody flag), however, it seems like it tries to set the player down, but because the player could theoretically move, the server corrects his position or sth:
![image](https://user-images.githubusercontent.com/39070261/95321931-08c54b00-089c-11eb-9cfb-e94e7a223af7.png)
(you can see he's falling down a few ft, but then suddenly the position is correct again)
- Generally, as soon as the animation is canceled or the player moves, he will appear on the correct position and won't fall back down.
- You can fix the players position with setEntityCoords on the client that streams in, but you need to wait until the "fall down"-process is done. The player won't fall down again or at least not as long as you don't leave his stream range again. The animation won't get canceled, it will only cancel if you set the position on serverside.

Example code to watch the "fall down"-process:
```
let PosTest = 0;
function LogPosition(entity){
    alt.setTimeout(() => {
        PosTest++;
        alt.log(entity.pos);
        if(PosTest < 15)
            LogPosition(entity);
    }, 500);
}

alt.on('gameEntityCreate', (entity) => {
  if(entity instanceof alt.Player) {
    LogPosition(entity)
  }
});
```

Maybe this will help reproducing and fixing this issue!

**Comment by martonp96** [comment]
The latest dev build (dev/2.1-dev7) should fix this issue, please try it and provide feedback if you can.

**Comment by Sushi1337** [comment]
@martonp96 Occurred within 15 minutes before the fix, now couldn't reproduce it within an hour of testing.
Seems to be fixed 👍

**Comment by martonp96** [comment]
fixed since dev/2.1-dev12
